examples/default/realign.json

```diff
@@ -0,0 +1,37 @@
+{
+    "@context": 
+      {
+        "bids" : "https://some/url/to/bids_context.jsonld",
+        "exp_dir" : {"@id" : "$HOME/nidm-results-examples/spm-default/ds011/"}
+      },
+    "@id": "http://example.org/ds00000X",
+    "generatedAt": "2020-03-10T10:00:00",
+    "wasGeneratedBy": {
+        "@id": "INRIA",
+        "@type": "Project",
+        "startedAt": "2016-09-01T10:00:00",
+        "wasAssociatedWith": { "@id": "NIH",
+                               "@type": "Organization",
+                               "hadRole": "Funding"
+                             }
+      },
+    "@graph": [
+      {
+        "@id" : "realign",
+        "@type" : "bids:bids_activity",
+        "wasAssociatedWith" : "bids:spmspatial.realign.estwrite",
+        "startedAtTime" : "10/10/2020 00:00:00",
```

Comment:
This sounds good. We could then have the type defined in the W3C PROV JSON context (as in https://www.w3.org/TR/json-ld11/#example-59-expanded-term-definition-with-type-coercion) rather than explicitely stated.